src/OpenTelemetry.Exporter.Console/ConsoleActivityExporter.cs

```diff
@@ -71,9 +71,7 @@ public override ExportResult Export(in Batch<Activity> batch)
                             continue;
                         }
 
-                        var array = tag.Value as Array;
-
-                        if (array == null)
+                        if (tag.Value is not Array array)
```

Comment:
I didn't know this before (that `is not` can be associated with a local variable), learned something new today! 😆